rasa/shared/core/domain.py

```diff
@@ -721,11 +741,18 @@ def __init__(
             session_config: Configuration for conversation sessions. Conversations are
                 restarted at the end of a session.
         """
-        self.entities, self.roles, self.groups = self.collect_entity_properties(
-            entities
-        )
+        (
+            self.entities,
+            self.roles,
+            self.groups,
+            self.default_entity_featurization,
```

Comment:
I would actually label the attribute as `self.default_ignored_entities` the same as you used as parameter in the different method signatures; it's clearer what it contains.

Reply:
Ah ja that makes sense I think this was an accidental leftover from a first attempt